src/components/Transitions/Fade.js

```diff
@@ -0,0 +1,121 @@
+import React, { PureComponent } from 'react'
+import Transition from 'react-transition-group/Transition'
+import PropTypes from 'prop-types'
+
+const TIMEOUT = 0
+const TRASITION_TIME = 400
+const START_OPACITY = 0
+const END_OPACITY = 1
+
+const OPACITY_DEFAULT = {
+  start: 0,
+  end: 1,
+}
+
+const AXIS = 'y'
+
+class Fade extends PureComponent {
+  static propTypes = {
+    /** Delay in milliseconds until animation start. */
+    timeout: PropTypes.number,
+
+    /** Time of animation in milliseconds. */
+    transitionTime: PropTypes.number,
+
+    /**
+     * Settings for opacity during animation.
+     *
+     * Default start: 0
+     * Default end: 1500 (If content's height is larger than 1500, pass the content's height here )
+    */
+    opacity: PropTypes.shape({
+      start: PropTypes.number,
+      end: PropTypes.number,
+    }),
+
+    /** Axis that is animated */
+    axis: PropTypes.oneOf(['x', 'y']),
+
+    /** Show the component; triggers the enter or exit states */
+    in: PropTypes.bool,
+
+    /**
+     * A convenience prop that enables or disabled appear animations for
+     * all children. Note that specifying this will override any defaults
+     * set on individual children Transitions.
+     */
+    appear: PropTypes.bool,
+
+    /** Optional style overrides. */
+    style: PropTypes.object,
+  }
+
+  static defaultProps = {
+    timeout: TIMEOUT,
+    transitionTime: TRASITION_TIME,
+    opacity: OPACITY_DEFAULT,
+    axis: AXIS,
+    style: {},
+    in: true,
+    appear: true,
+  }
+
+  get opacity() {
```

Comment:
Any particular reason for the getters? Not opposed to them at all, just curious :)

Reply:
I like the API and I think it encourages good separation of concerns